vmdb/spec/factories/container_node.rb

```diff
@@ -0,0 +1,5 @@
+FactoryGirl.define do
+  factory :container_node do
+    sequence (:name)  { |name| "container_node_#{seq_padded_for_sorting(name)}" }
```

Comment:
I'm gonna kill the specs altogether, since it's just testing AR like you noted above.